native/explorer/src/series.rs

```diff
@@ -749,3 +749,24 @@ pub fn s_coalesce(data: ExSeries, other: ExSeries) -> Result<ExSeries, ExplorerE
     let coalesced = s1.zip_with(&s1.is_not_null(), s2)?;
     Ok(ExSeries::new(coalesced))
 }
+
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn s_is_in(data: ExSeries, rhs: ExSeries) -> Result<ExSeries, ExplorerError> {
+    let s: &Series = &data.resource.0;
+    let rhs: &Series = &rhs.resource.0;
+
+    let s = match s.dtype() {
+        DataType::Int8
+        | DataType::UInt8
+        | DataType::Int16
+        | DataType::UInt16
+        | DataType::Int32
+        | DataType::UInt32
+        | DataType::Int64 => s.i64().unwrap().is_in(rhs)?,
+        DataType::Float32 | DataType::Float64 => s.f64().unwrap().is_in(rhs)?,
+        DataType::Utf8 => s.utf8().unwrap().is_in(rhs)?,
+        dt => panic!("is_in/2 not implemented for {:?}", dt),
```

Comment:
Do we need to unwrap? Could we do `s.is_in(rhs)` directly?

Reply:
Maybe `equal` could be used as reference here too.

Reply:
I don'tit is possible to do directly, the compiler type-checker returns an error and suggests using `s.utf8()?.is_in(rhs)?` . That happens because `i64/f64/uft8` returns a Result. 
I can change to the `?` operator, though.

Reply:
I see. I think we should include date/datetime comparisons then. Or is that not possible?

Reply:
I don't see any reason why it wouldn't be. Let me try.